packages/components/src/components/FunctionChart.tsx

```diff
@@ -70,81 +172,21 @@ export const FunctionChart: React.FC<FunctionChartProps> = ({
     ) : (
       <></>
     );
-  let data1 = _rangeByCount(
-    chartSettings.start,
-    chartSettings.stop,
-    chartSettings.count
-  );
-  type point = { x: number; value: result<Distribution, string> };
-  let valueData: point[] = React.useMemo(
-    () =>
-      data1.map((x) => {
-        let result = runForeign(fn, [x], environment);
-        if (result.tag === "Ok") {
-          if (result.value.tag == "distribution") {
-            return { x, value: { tag: "Ok", value: result.value.value } };
-          } else {
-            return {
-              x,
-              value: {
-                tag: "Error",
-                value:
-                  "Cannot currently render functions that don't return distributions",
-              },
-            };
-          }
-        } else {
-          return {
-            x,
-            value: { tag: "Error", value: errorValueToString(result.value) },
-          };
-        }
-      }),
+
+  let _getPercentiles = React.useMemo(
```

Comment:
Why the underscore naming here? I currently can't actually think of a better name but starting a variable with an underscore usually means it's not used to me